source/resources/examples/typescript/restriction/GettingAccountRestrictions.ts

```diff
@@ -28,11 +28,11 @@ const repositoryFactory = new RepositoryFactoryHttp(nodeUrl);
 const restrictionHttp = repositoryFactory.createRestrictionAccountRepository();
 
 restrictionHttp.getAccountRestrictions(address)
-    .subscribe((accountRestrictions) => {
+    .subscribe((accountRestrictions: any) => {
```

Comment:
Could it be the actual type rather than `any`?

Reply:
What is the actual type? Typescript is still a bit obscure to me.

Reply:
so it should be the method's return type. `AccountRestrictions`

Reply:
I'm afraid my typescript is not strong enough. I have tried with `AccountRestrictions` and `AccountRestrictions[]` and both failed. Unless you give me more hints, I think I'll use `any` (which is what `symbol-cli` does).

Reply:
it should be `AccountRestrictions` as the ts sdk defined. what errors have you got?

Reply:
After changing `any` to `AccountRestrictions` and adding `AccountRestrictions` to the import line:
```
restriction/GettingAccountRestrictions.ts:32:29 - error TS2339: Property 'length' does not exist on type 'AccountRestrictions'.

32     if (accountRestrictions.length > 0) {
                               ~~~~~~

restriction/GettingAccountRestrictions.ts:34:10 - error TS2339: Property 'filter' does not exist on type 'AccountRestrictions'.

34         .filter((accountRestriction: any) => accountRestriction.values.length > 0)
            ~~~~~~
```